OrbitQt/orbitmainwindow.h

```diff
@@ -98,7 +99,8 @@ class OrbitMainWindow : public QMainWindow {
   QIcon icon_start_capture_;
   QIcon icon_stop_capture_;
 
-  std::string m_CurrentPdbName;
```

Comment:
Was this intentionally removed, seems quite unrelated to the PR?
Apparently it was unneeded, so it's fine for me, just want to make sure.

Reply:
Not related no - but since it was unused I removed it.